Cargo.toml

```diff
@@ -110,6 +110,9 @@ vlog = "0.1.4"
 mime = "0.3.13"
 farmhash = "1.1.5"
 failure = "0.1"
+rustls = "0.16"
+tokio-rustls = "0.10"
```

Comment:
Why not use openssl, it helps compile time since it's already in the dependency tree.

```toml
openssl = "0.10"
tokio-openssl = "0.2"
```